src/Microsoft.FeatureManagement/FeatureFilters/TimeWindowFilter.cs

```diff
@@ -33,7 +37,26 @@ public TimeWindowFilter(ILoggerFactory loggerFactory)
         /// <returns><see cref="TimeWindowFilterSettings"/> that can later be used in feature evaluation.</returns>
         public object BindParameters(IConfiguration filterParameters)
         {
-            return filterParameters.Get<TimeWindowFilterSettings>() ?? new TimeWindowFilterSettings();
+            DateTimeOffset? Start = filterParameters["Start"] != null ? filterParameters.GetValue<DateTimeOffset>("Start") : (DateTimeOffset?)null;
+            DateTimeOffset? End = filterParameters["End"] != null ? filterParameters.GetValue<DateTimeOffset>("End") : (DateTimeOffset?)null;
+
+            TimeWindowFilterSettings settings = new TimeWindowFilterSettings()
+            {
+                Start = Start,
+                End = End
+            };
+
+            List<string> Filters = filterParameters.GetSection("Filters").Get<List<string>>();
```

Comment:
local variables should use lowercase param names.


```suggestion
            IEnumerable<string> filters = filterParameters.GetSection("Filters").Get<IEnumerable<string>>();
```